documentation/bin/createDocu.groovy

```diff
@@ -72,7 +72,7 @@ class TemplateHelper {
 
         parameters.keySet().toSorted().each {
             def props = parameters.get(it)
-            t += "| `${it}` | ${props.GENERAL_CONFIG ? 'X' : ''} | ${props.STEP_CONFIG ? 'X' : ''} | ${props.STAGE_CONFIG ? 'X' : ''} |\n"
+            t += "| `${it}` | ${props.GENERAL_CONFIG ? 'X' : ''} | ${props.STEP_CONFIG ? 'X' : ''} | ${props.STAGE_CONFIG ? 'X' : ''} |\n"
```

Comment:
What has been changed here. I see one whitespace replaced by another whitespace. In case this is not relevent I would like to suggest to remove this file from the PR in order to keep the history clear.

Reply:
Now it's a *real* whitespace.
I didn't investigate which char it was before, but my IDE didn't display it as normal whitespace.